src/guc.c

```diff
@@ -997,6 +998,17 @@ _guc_init(void)
 							 NULL,
 							 NULL);
 
+	DefineCustomBoolVariable(MAKE_EXTOPTION("enable_sparse_index_bloom1"),
```

Comment:
Why do we call this bloom1? Is there a specific reason?

Reply:
There's a good chance we'll have to break binary compatibility in the next version, and it will be `bloom2` :) Maybe we should have encoded it into the bloom filter value, but I did it the ugly way for simplicity and space saving.